module/User/src/Model/Enums/ApiPermissions.php

```diff
@@ -4,7 +4,7 @@
 
 namespace User\Model\Enums;
 
-use Laminas\Mvc\I18n\Translator;
+use Laminas\I18n\Translator\TranslatorInterface;
```

Comment:
Same as I mentioned earlier about the translator.

Reply:
b48bbb2e9966b14482d8bd2b22ee043892724aac